lib/systems/platforms.nix

```diff
@@ -377,6 +377,13 @@ rec {
     };
   };
 
+  apple-m1 = {
+    gcc = {
+      arch = "armv8.3-a+crypto+sha2+aes+crc+fp16+lse+simd+ras+rdm+rcpc";
+      cpu = "apple-a13";
```

Comment:
GCC doesn't like either of these things:

* https://gcc.gnu.org/onlinedocs/gcc/AArch64-Options.html

But we force them in wrappers. Is there a better way?

Reply:
Huh, that is weird. @thefloweringash does this help somewhere else?

Reply:
Yes: https://github.com/NixOS/nixpkgs/pull/105026#issuecomment-774757005